### ISSUE TITLE
DB password should not be mandatory (if postgres credential uses the unix account directly)

### ISSUE THREAD

**Issue by sbodmer** [3dcitydb/importer-exporter#30]
On Debian, it's possible to create a postgres user without password, the authentication is done directly with the unix credential (ex: if the user "citydb" starts pgsql and the postgres user "citydb" exists, then the
user can have acces to his database directly).

The java importer/exporter **imposes** to enter a db password, the password should be **optional** in this case.

**Comment by clausnagel** [comment]
A database connection password is now optional (1966a46ab868efb4fa82147ec161dfdbd499c74c).

Can you please verify whether this commit already solves your issue? Simply download the source and run the build.xml with Ant to compile the Importer/Exporter tool.

**Comment by sbodmer** [comment]
Hi Claus, thanks for your speedy feedback.

I downloaded your patch and tested it, but sadly it will not solve my problem but I think I know why...
The error I get is
`The server requested password-based authentication, but no password was provided`

After some digging I found out that I use the "peer" authentication which uses unix socket to talk
to the linux kernel to get the credential. The Java JDBC driver only supports TCP connection (which needs a password), so the authentication without password will not work in this context (until a native JDBC is available...)
[Postgres JDBC doc](https://jdbc.postgresql.org/documentation/head/prepare.html)

Sorry for the trouble, and thanks for your work in any case.
